tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -134,10 +134,10 @@ Has Spawn Failed
 
 Spawn Notebook With Arguments  # robocop: disable
    [Documentation]  Selects required settings and spawns a notebook pod. If it fails due to timeout or other issue
-   ...              It will try again ${retries} times (Default: 1). Environment variables can be passed in as kwargs
-   ...              By creating a dictionary beforehand
+   ...              It will try again ${retries} times (Default: 1) after ${retries_delay} delay (Default: 0 seconds).
+   ...              Environment variables can be passed in as kwargs by creating a dictionary beforehand
    ...              e.g. &{test-dict}  Create Dictionary  name=robot  password=secret
-   [Arguments]  ${retries}=1  ${image}=s2i-generic-data-science-notebook  ${size}=Small
+   [Arguments]  ${retries}=1  ${retries_delay}=0 seconds  ${image}=s2i-generic-data-science-notebook  ${size}=Small
```

Comment:
_Verify Pytorch And Tensorflow Can Be Spawned_ failed because it is calling the keyword `Verify Image Can Be Spawned` (defined in tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot), which is calling  `Spawn Notebook With Arguments` using positional arguments instead of named arguments

I see two possible solutions:
- Define the new ${retries_delay} argument at the end, so it doesn't break existing code
- Review and fix all calls to `Spawn Notebook With Arguments` not to have this problem

To me it's fine if you do the fist one

Reply:
I had a quick at the locations where `Spawn Notebook With Arguments` is called, and I could only find one place (`Verify Image Can Be Spawned`) where the keyword is called with positional arguments, so I'll change it there

Reply:
should `Verify Image Can Be Spawned` accept the `retries_delay` ? 🤔

Reply:
>I had a quick at the locations where Spawn Notebook With Arguments is called, and I could only find one place (Verify Image Can Be Spawned) where the keyword is called with positional arguments, so I'll change it there
I've searched for it too and also found it just in one place

>should Verify Image Can Be Spawned accept the retries_delay ?
ok for me. I think that keyword is only used in `Verify Pytorch And Tensorflow Can Be Spawned`

Reply:
ok for you ... to accept it? or not to accept it (current code in https://github.com/red-hat-data-services/ods-ci/pull/457/commits/f6b0528b4b547fd9d7e05dab7eed3252ec59861b)

Reply:
IMHO it should accept it, even though it doesn't currently use it it could come in handy down the line

Reply:
updated as suggested